schema/user.js

```diff
@@ -29,7 +30,11 @@ const userResolvers = {
         v1AccessToken,
         v2AccessToken
       );
-      return { userSettings };
+      const currencies = await dataSources.walloraAPI.getCurrencies(
+        v1AccessToken,
+        v2AccessToken
+      );
+      return { userSettings, "currencies" : currencies };
```

Comment:
Replace `"currencies" : currencies` with just `currencies`

Reply:
currencies variable is an array hence can't be used directly

Reply:
added